frontend_tests/zjsunit/index.js

```diff
@@ -15,6 +15,7 @@ global.assert = require('assert');
 require('core-js/features/string/code-point-at');
 
 global._ = require('underscore/underscore.js');
+global.autosize = require('autosize/dist/autosize.js');
```

Comment:
I think you should be requiring this in the individual test files that use it; we only do jquery globally.

Reply:
Or better, see if you can remove it from all global-related locations (`.eslintrc.json`, `frontend_tests`, `static/js/bundles`, `tools/webpack.config.ts`), and instead just `require()` it as a module in the files where it’s actually used (`static/js/{compose_actions,compose_ui,composebox_typeahead,invite,message_list,resize}.js`).

Reply:
Made both of these changes and everything seems to be working fine on manual testing as well.